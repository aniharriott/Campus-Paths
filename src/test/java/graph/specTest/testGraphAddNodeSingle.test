# Test Graph addNode single file.
# Tests if a singular node can
# be added to the graph
# successfully.

# Create graph
CreateGraph graph1

# Add node
AddNode graph1 n1