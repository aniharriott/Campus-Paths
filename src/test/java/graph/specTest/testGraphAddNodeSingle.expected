# Test Graph addNode single file.
# Tests if a singular node can
# be added to the graph
# successfully.

# Create graph
created graph graph1

# Add node
added node n1 to graph1